src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -49,11 +51,37 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
 
 Access to log management capabilities is limited by user type. See [user type capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type#user-type-capabilities).
 
-## Set up errors inbox [#get-started]
+## Use errors inbox from entity scoped views [#scoped]
+
+Entity scoped views allow you to view errors for a single APM, browser, mobile, or [Open Telemetry](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/#benefits) application. Access the errors inbox view for a specific entity by navigating to that entity in the browser or mobile monitoring view and selecting **errors inbox** in the left nav under **triage**. 
```

Comment:
I still like the the idea of having the word Entity link to: https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/
but if we think that is a distraction from the main point I am ok leaving it out.

Reply:
We try not to link out to "basic" NR concepts such as entity, but I believe that it does make sense in this specific situation. I'll add a link.

Reply:
+1 for that. We often assume our users known these definitions but it doesn't hurt to clarify.

Reply:
@obarney - the style docs ask that we refer to the product as errors inbox when used in the middle of a sentence https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/